frontend/src/components/static_pages/about.css

```diff
@@ -1,154 +1,146 @@
+.container {
```

Comment:
this class is used everywhere and changing it here does affect where it is used elsewhere in the application.

We need to make this change just for the `about`

I am not sure this change is necessary as it mostly getting overwritten by other styles elsewhere

Reply:
@dkaplan88, this is in the about.css file and is only imported by the about.jsx file so it is scoped to only alter this page. 

I agree that it can be very dangerous to add css rules globally defined classnames.

Reply:
I was incorrect, this doesn't scope to that page. You could just add a `className` to the container of the about page and reference that class name here.

Reply:
Hi!!!  
I fixed the problem and used a className as Jim suggested, I named it mainContainer, I hope this fixes the problem... if not let me know!! 

Thank you :)